### ISSUE TITLE
Notes source

### ISSUE THREAD

**Issue by cup** [mathiasbynens/jsesc#49]
Sorry if this is off topic

i see this page

https://mathiasbynens.be/notes/javascript-escapes

however i do not see a repo for those posts

do you have some "notes" repo somewhere?

**Comment by mathiasbynens** [comment]
No. Why are you asking? Would you like to suggest a correction?

**Comment by cup** [comment]
I like the website style and maybe others would like to use the source for their benefit
